src/components/mainPage/MainPage.css

```diff
@@ -0,0 +1,14 @@
+
+@media only screen and (min-width: 750px  ) {
+
+    .main_page_container {
+        margin: 3%;
```

Comment:
This will change all margins and depend heavily on parent (% of parent). if you want relative units, use vw instead. For y axis margin, use fixed values (px) since y axis dimensions don't depend on screen size that much.

Reply:
I fixed this and I using px now.